src/elements/common/back-button/BackButton.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @flow
+ * @file Back Button component
+ * @author Box
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { withRouter } from 'react-router-dom';
+import type { RouterHistory } from 'react-router-dom';
+import IconNavigateLeft from '../../../icons/general/IconNavigateLeft';
```

Comment:
Thoughts on using https://opensource.box.com/box-ui-elements/#/Icons?id=iconpageback?

Reply:
It looks like that icon is intended to be used with pagination controls. The other back buttons in EUA use the IconNavigateLeft icon.